### ISSUE TITLE
chairman speeches do not contain who attribute

### ISSUE THREAD

**Issue by matyaskopp** [calzada/PARLAMINT-ES-MC#24]
The chairman's speeches do not refer to a person that speaks:
https://github.com/calzada/PARLAMINT-ES-MC/blob/7d8412564b9686b396376d33f5cd9befb009f3c4/ParlaMint.sample/ParlaMint-ES_2015-01-20-CD150120.xml#L105-L108

the information is present in the source:
https://github.com/calzada/PARLAMINT-ES-MC/blob/7d8412564b9686b396376d33f5cd9befb009f3c4/CD.sample/CD150120.xml#L52

[cd2parmamint.xsl](https://github.com/calzada/PARLAMINT-ES-MC/blob/master/bin/cd2parmamint.xsl) needs to be improved:
- [ ] adding chairman in who attribute and also in a listPerson in temporary component files

[parlamint2root.xsl](https://github.com/calzada/PARLAMINT-ES-MC/blob/master/bin/parlamint2root.xsl) needs changes too:
- [ ] the chairing person does not contain party affiliation in the source, but in other meetings, he/she can be presented as a regular deputy (then the information is present)

reported here: https://github.com/clarin-eric/ParlaMint/issues/696#issue-1765368729

**Comment by rdelibanoc** [comment]
@matyaskopp we need confirmation of this. How do we proceed to fix this error, do we change it in the source, or shall we change it in the Tei xml. I think we should change it in the source, does this mean we run the markup and annotation process from scratch.

**Comment by matyaskopp** [comment]
- We don't want to change the source CD files, it is defined in cd.dtd file and no changes are allowed (@calzada)
- We don't want to change it manually in TEI files - they are automatically generated, so the manual changes will be destroyed with the next update

We want to change the scripts, as I described in the issue.
- if there is not enough information in CD files, then another additional source can be added


You can start with data that is present in CD format:
- parse name
- generate a person id (in the same way other ids are generated)
- create a `<person>` element that contains only `person/@xml:id` and `person/persName`(+ child nodes)


---
side note: The general idea of adding new sources  that I used in adding government members is

- download data.  eg: https://github.com/calzada/PARLAMINT-ES-MC/blob/be3e2be3cf70619c4cd8513b90b19df4d54db87d/Makefile#L88-L95
- add them to the git repository (in case someone changes the source)
- process/use the data: https://github.com/calzada/PARLAMINT-ES-MC/blob/be3e2be3cf70619c4cd8513b90b19df4d54db87d/Makefile#L97-L98

**Comment by matyaskopp** [comment]
the name is in the `<chair>` element and is the same for the whole meeting I guess:
https://github.com/calzada/PARLAMINT-ES-MC/blob/7d8412564b9686b396376d33f5cd9befb009f3c4/CD.sample/CD150120.xml#L52

**Comment by calzada** [comment]
Yes but we cannot acceso to the name since our transcripts only soy MR
PRESIDENT without further details.
Best
Mc

**Comment by matyaskopp** [comment]
so I have no idea what
```XML
<chair who="JESÚS POSADA MORENO"> 
```
means in CD files. According to wikipedia, Jesús Posada was a chairman...

**Comment by calzada** [comment]
Yes but he start the sesión as chairman  and the chair may change in the
middle of it. This is why it is not reliable. We could easily add this
person as chair but it May turn out the chair is not always him.
Best
Mc

**Comment by matyaskopp** [comment]
element `body` allows multiple `chair` elements
https://github.com/calzada/PARLAMINT-ES-MC/blob/be3e2be3cf70619c4cd8513b90b19df4d54db87d/CD.sample/cd.dtd#L26
 so I expected that if the speaker change happened, there would be a new chair element.

I have checked your source CD files, and only one `chair` element exists. So you are claiming there that there is no chairman change. If we use CD files as a source, it is probably better to propagate this error (if it is an error, there are no chairman changes).

If there is no chairman change in source pdf(?) files, then you should expect that there was no chairman change in the chamber of deputies.

**Comment by calzada** [comment]
Dear Matyas,
This dtd applies to all our working parliaments (CD, EP, HC). But since
there are divergencies between and among parliaments, the dtd must
accommodate for all of them.
Best,
mc

**Comment by matyaskopp** [comment]
Can you give me a sample of chairman changes? I have found only a few, where PRESIDENTA changes to PRESIDENTE, but it looks more like a typo in the source (wrong gender). 
eg https://github.com/calzada/PARLAMINT-ES-MC/blob/be3e2be3cf70619c4cd8513b90b19df4d54db87d/CD/CD230214.xml
there are these chairs:
```
53  PRESIDENTA,UNKNOWN
1   PRESIDENTE,UNKNOWN
16  VICEPRESIDENTA,Elizo Serrano, María Gloria
13  VICEPRESIDENTA,Pastor Julián, Ana María
```
There is only one occurrence of PRESIDENTE, so we can probably say it is a typo and set `MERITXELL BATET LAMAÑA`
https://github.com/calzada/PARLAMINT-ES-MC/blob/be3e2be3cf70619c4cd8513b90b19df4d54db87d/CD/CD230214.xml#L69
as chair.
But I can be wrong - the idea is to record what is in the transcriptions - we can say it is truth...

**Comment by calzada** [comment]
It is not a typo. It is a situation where the chair changes. Regarding
VICEPRESIDENTS it is not quite sure the take the chair, although they may
have taken it.
Best
mc

**Comment by calzada** [comment]
@matyaskopp 
You could equally use "UNKNOWN". This is what we did at ECPC (our research group.
